plugins/vertica/database_credentials_test.go

```diff
@@ -0,0 +1,32 @@
+package vertica
+
+import (
+	"testing"
+	
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+	
+func TestDatabaseCredentialsProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, DatabaseCredentials().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"default": {
+			ItemFields: map[sdk.FieldName]string{
+				fieldname.User: "vertica",
+				fieldname.Password: "",
+				fieldname.Host: "localhost",
```

Comment:
why is the password empty in this test case?

Reply:
updated